fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -447,9 +206,7 @@ public T run(@Nonnull Function<? super FDBRecordContext, ? extends T> retriable)
      * @return result of function after successful run and commit
      * @see #runAsync(Function)
      */
-    public <T> T run(@Nonnull Function<? super FDBRecordContext, ? extends T> retriable) {
-        return run(retriable, null);
-    }
+    <T> T run(@Nonnull Function<? super FDBRecordContext, ? extends T> retriable);
```

Comment:
Does this make sense to leave as a `default` method that still calls the other `run` method? Similarly for some of the `runAsync` methods that essentially supply the default values for optional parameters.